Improve docs (#14)
yiisoft/hydrator-validator: README.md

## General usage

> Validating hydrator is a decorator for [another hydrator](https://github.com/yiisoft/hydrator). It allows to validate
> raw data before passing it to the decorated hydrator and to validate object after creating or populating.

Reviewer: ```suggestion
Validating hydrator is a [hydrator](https://github.com/yiisoft/hydrator) decorator that allows to validate
raw data before passing it to the decorated hydrator and to validate object after creating or populating it.
```
